Firestore/core/src/firebase/firestore/remote/exponential_backoff.h

```diff
@@ -107,6 +107,7 @@ class ExponentialBackoff {
   const Milliseconds initial_delay_;
   const Milliseconds max_delay_;
   util::SecureRandom secure_random_;
+  std::chrono::time_point<std::chrono::steady_clock> last_attempt_time_;
```

Comment:
Totally optional: sorry, forgot that `steady_clock` has a nested alias for this, which is _slightly_ shorter:
```cpp
std::chrono::steady_clock::time_point last_attempt_time_;
```
Feel free to disregard.

Reply:
Ah, nice!